bool stop_sign
bool traffic_light
string move
